src/uikit.cs

```diff
@@ -14720,12 +14724,15 @@ partial interface UIPrinterPickerController {
 		[Static, Export ("printerPickerControllerWithInitiallySelectedPrinter:")]
 		UIPrinterPickerController FromPrinter ([NullAllowed] UIPrinter printer);
 	
+		[Async (ResultTypeName = "UIPrinterPickerCompletionResult")]
 		[Export ("presentAnimated:completionHandler:")]
 		bool Present (bool animated, UIPrinterPickerCompletionHandler completion);
 	
+		[Async (ResultTypeName = "UIPrinterPickerCompletionResult")]
 		[Export ("presentFromRect:inView:animated:completionHandler:")]
 		bool PresentFromRect (CGRect rect, UIView view, bool animated, UIPrinterPickerCompletionHandler completion);
 	
+		[Async (ResultTypeName = "UIPrinterPickerCompletionResult")]
```

Comment:
I do not think we can Async'ify this one according to [docs](https://developer.xamarin.com/guides/cross-platform/macios/binding/binding-types-reference/#AsyncAttribute):

> You can use the `[Async]` attribute on methods that **return void** and whose **last** argument is a callback.